test/utils/TestUtils.ts

```diff
@@ -628,3 +641,5 @@ export type {
   SupportedType,
   SupportedDeployVerifier,
 };
+
+// delete prepareRelayTransaction
```

Comment:
@franciscotobar Is this comment still valid?

Reply:
Updated